src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/CAS3SubjectAccessRequestServiceTest.kt

```diff
@@ -41,6 +41,28 @@ class CAS3SubjectAccessRequestServiceTest : SubjectAccessRequestServiceTestBase(
     )
   }
 
+  @Test
+  fun `Get CAS3 Information - Null dates check`() {
+    val (offenderDetails, _) = `Given an Offender`()
+    val result =
+      sarService.getCAS3Result(offenderDetails.otherIds.crn, offenderDetails.otherIds.nomsNumber, null, null)
+    assertJsonEquals(
+      """ 
+     {
+         "Applications": [ ],
+         "Assessments": [ ],
+         "AssessmentReferralHistoryNotes" : [ ],
+         "Bookings": [ ],
+         "BookingExtensions": [ ],
+         "Cancellations": [ ],
+         "DomainEvents": [ ],
+         "DomainEventsMetadata": [ ]
```

Comment:
just checking an assumption - the intention of these new checks are to make sure the code doesn't fall over when dates aren't defined, and if data was defined we'd be seeing that here (but to keep things simple you haven't added any test data for these tests)

Reply:
that's correct. could have made more tests for this, just covering off the scenario i found in testing yesterday.